README.md

```diff
@@ -82,8 +82,9 @@ Other options include:
   -e, --encoding=encoding      [default: utf8] text encoding to use
   -f, --format=json|stylish    [default: stylish] formatter to use for outputting results
   -h, --help                   show CLI help
-  -m, --maxResults=maxResults  [default: all] maximum results to show
+  -max=max                     [default: all] maximum results to show
```

Comment:
I believe `maxResults` was a better choice, as `max` is too ambiguous and may become troublesome in future.
For instance, quite a few linters provide a way to set a maximum amount of warnings allowed.
Examples: